### PR TITLE
Fix in the layout - wont be too visible without the images tho

### PR DESCRIPTION
**Pull request by cyntss**

### CODE REVIEW

**Comment by cyntss** [comment]
I Need to fix those tests

File: app/views/dashboard/index.html.haml [hunk 1]
```diff
@@ -1,35 +1,35 @@
...
+                  Product:
+                  = link_to( 'SUSE Customer Center', 'https://scc.suse.com/login')
+            .col-sm-6
+              %ul
+                %li
+                  %i.fa.fa-medkit.fa-fw
+                  Teamlead: Thomas Schmidt
+                %li
+                  %i.fa.fa-envelope.fa-fw
+                  Email: happy-customer@suse.de
```
Comment (thutterer) [inline app/views/dashboard/index.html.haml]: why not make this a mailto link?

File: app/views/dashboard/index.html.haml [hunk 2]
```diff
@@ -69,7 +69,7 @@
                   = user.birthday.strftime("%d")
                   —
```
Comment (thutterer) [inline app/views/dashboard/index.html.haml]: while you are at it, this looks broken when more than one person has a birthday this month.
currently visible in our production dashboard

**Comment by NadjaW** [comment]
@cyntss can we use another background picture? i dont like the colors and the style, maybe we can talk about that in our teamboard stand up tomorrow.

**Comment by thutterer** [comment]
I fixed the photo's aspect ratio in cbe9160 ... that was the simpler and more flexible fix. now works with all ratios, even portrait. And @NadjaW and me agreed on not liking the circled image and the background panel too much anyway.
But I don't know if you still want to change the column sizes of the other panels. Then please re-open after updating.

**Pull request closed without merging**